roles/ovb/bmc/tasks/deploy_files.yml

```diff
@@ -0,0 +1,9 @@
+- name: Deploy config-bmc-ips
+  copy:
+      src="config-bmc-ips.service"
```

Comment:
It's probably a new thing that it works as for ansible < 2.0 it would have failed...
I would rather have a consistent way of doing things, currently, all playbooks use the ":" syntax, please align to avoid confusion and until we have the styling guide in place